Move System.Diagnostics.Process lengthy remarks to files (#5360)
dotnet/dotnet-api-docs: includes/remarks/System.Diagnostics/Process/ExitCode.md

Use <xref:System.Diagnostics.Process.ExitCode%2A> to get the status that the system process returned when it exited. You can use the exit code much like an integer return value from a `main()` procedure.

The <xref:System.Diagnostics.Process.ExitCode%2A> value for a process reflects the specific convention implemented by the application developer for that process. If you use the exit code value to make decisions in your code, be sure that you know the exit code convention used by the application process.

Developers usually indicate a successful exit by an <xref:System.Diagnostics.Process.ExitCode%2A> value of zero, and designate errors by nonzero values that the calling method can use to identify the cause of an abnormal process termination. It is not necessary to follow these guidelines, but they are the convention.

If you try to get the <xref:System.Diagnostics.Process.ExitCode%2A> before the process has exited, the attempt throws an exception. Examine the <xref:System.Diagnostics.Process.HasExited%2A> property first to verify whether the associated process has terminated.

> [!NOTE]

> >  When standard output has been redirected to asynchronous event handlers, it is possible that output processing will not have completed when <xref:System.Diagnostics.Process.HasExited%2A> returns `true`. To ensure that asynchronous event handling has been completed, call the <xref:System.Diagnostics.Process.WaitForExit> overload that takes no parameter before checking <xref:System.Diagnostics.Process.HasExited%2A>.

Reviewer: ```suggestion
> When standard output has been redirected to asynchronous event handlers, it's possible that output processing will not have completed when <xref:System.Diagnostics.Process.HasExited%2A> returns `true`. To ensure that asynchronous event handling has been completed, call the <xref:System.Diagnostics.Process.WaitForExit> overload that takes no arguments before checking <xref:System.Diagnostics.Process.HasExited%2A>.
```
